daemon/cmd/api_handlers.go

```diff
@@ -106,56 +106,61 @@ func ciliumAPIHandlers(dp promise.Promise[*Daemon], cfg *option.DaemonConfig, _
 	// /healthz/
 	out.DaemonGetHealthzHandler = wrapAPIHandler(dp, getHealthzHandler)
 
+	// /service/
+	out.ServiceGetServiceHandler = wrapAPIHandler(dp, getServiceHandler)
+
+	// /service/{id}/
+	out.ServiceGetServiceIDHandler = wrapAPIHandler(dp, getServiceIDHandler)
+	out.ServiceDeleteServiceIDHandler = wrapAPIHandler(dp, deleteServiceIDHandler)
+	out.ServicePutServiceIDHandler = wrapAPIHandler(dp, putServiceIDHandler)
+
+	// If no control plane is set, we only expose service API to the user for programming.
+	if option.Config.LoadBalancerNoControlPlane {
```

Comment:
> we only expose service API to the user for programming

btw: this function only adds the api handlers that are still implemented "in the daemon" there are also other api handlers that are already provided by the respective cells directly.

As more and more of the api handlers will be migrated into the respective cells it would be actually nice to keep the list of api handlers that don't work if the K8s CP isn't available.

Reply:
Ok, fixed by enabling all + added context to why this is ok. Thanks for pointing this out!